olive/passes/pytorch/capture_split_info.py

```diff
@@ -153,28 +153,31 @@ def split_using_cost_model(
         split_idx = 0
         split_bytes = 0
         node_idx = 0
+        mem_intensive = False
         for name, _ in model.load_model(cache_model=False).named_modules():
-            if name not in module_to_bytes or name in modules_to_exclude:
+            if name not in module_to_cost or name in modules_to_exclude or name.lower().endswith("dropout"):
```

Comment:
Any reason to skip dropout here?

Reply:
since dropouts are only used in train mode, they don't appear in the exported inference graph. So, I removed it to make the split mappings cleaner.